### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by Psiale**
Having just read about testing in Javascript in this project we learn how to implement our reading in the form of Jest testing.
In this project we all given the objective to create five functions and create five separate test scripts for those functions, these functions include: 
 - Capitalize: Capitalises a string
 - Reverse string: Reverses a string (amazing!) 
 - Calculator: Returns an object with the basic operations add, subtract, multiply, divide
 - [Caesar-cypher](http://practicalcryptography.com/ciphers/caesar-cipher/): Takes a string and shifts letters up by the given number 
 - Array analysis: Takes an array of numbers and returns an object with average, min, max, and length.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team 👋

**Nice work on this so far.**

**You have done quite well on this. Kudos 💪**

- [ ] However there are some changes to be made, please check comments in your code and fix them all.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: README.md [hunk 1]
```diff
@@ -1 +1,104 @@
...
+ - NPM
+
+**Linters & Quality Control**
+ - [ESlint](https://eslint.org/)
+ - [StyleLint](https://stylelint.io/)
+
+### 💾 Setup & Use
+
+#### Setup
+For the live site, [head here.](https://mtodo-list.netlify.app/)
```
Comment (mikenath223) [inline README.md]: - [x] Please remove this live link on your readme. This is a testing project therefore it shouldn't have one.

File: package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "file-loader": "^6.2.0",
     "html-webpack-plugin": "^4.5.0",
     "jest": "^26.6.1",
-    "preload-webpack-plugin": "^3.0.0-beta.4",
+    "preload-webpack-plugin": "^3.0.0-beta.3",
     "sass": "^1.27.0",
     "sass-loader": "^10.0.4",
     "style-loader": "^2.0.0",
```
Comment (mikenath223) [inline package.json]: - [x] There are a lot of unnecessary dependencies here, please remove them and leave **ONLY** jest.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: darkred;
+}
```
Comment (mikenath223) [inline style.css]: - [x] This ia s testing project you don't need a style.css file. Please remove it.

**Comment by mikenath223** [comment]
- [x] There are two index.js files that were not used. Please remove them all.

**Comment by mikenath223** [comment]
- [ ] Your capitalize function doesn't work as expected. Kindly fix it. For the single test case, you have the expected result should be `LOLZ` instead. Please fix this and add more tests.

**Comment by mikenath223** [comment]
- [x] Your ceaser function is expected to have tests that check wrapping of alphabets for example passing a string with this argument `(2, zyx)` should return `baz`. Please add tests like this and ensure they pass.

**Comment by mikenath223** [comment]
- [x] Also your single tests on ceasarCipher shouldn't return such a result it should return lowercase letters if the input string is lowercase and uppercase if the input is uppercase. Yours returns the wrong result. Kindly check and fix it.

**Comment by Rhelli** [comment]
> This PR is for Microverse TSE review

# PR # 2 - Jest Testing 🧪

Following the previous TSEs requests, we have modified our project to adhere more closely to the TOP requirement. The following changes have been made:

 - The redundant 'live site' link has been removed from the readme
 - Redundant index.js and CSS files have been removed
 - Redundant dependencies have been removed from this project only leaving Jest and ESLint
 - The cipher function has been fixed so it returns a string with characters in the same case
 - The cipher function no wraps within the alphabet if the letters number exceeds 26

# **Important**  ☝🏽 ❗

image

1. As per the screenshot from the Odin Project above (👆🏽), we haven't modified our capitalize function as it is currently working correctly and should only capitalize the first letter of the given string.

2. We were requested to remove the CSS stylesheet and only keep the Jest package, so we have also removed any Plugins or Loaders for processing CSS (as it is not at all required by this project). **However, this also creates an error due to Microverse's default `linters.yml` file, therefore, we have removed the Stylelint checks from this project**

Thanks in advance,

Alexis and Rory

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @Psiale and @Rhelli  :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by Rhelli** [comment]
🌟🍾 Excellent! Thanks to @mikenath223 and @MuriloRoque for your reviews!

**Pull request merged**